src/lib/update-alternatives/UI/main_dialog.rb

```diff
@@ -46,6 +46,24 @@ def dialog_content
       )
     end
 
+    def help_handler
+      Yast::Popup.LongText(
+        _("Help"),
+        RichText(
+          _("<p>The status of an alternative, can be automatic or manual. " \
+          "In <b>automatic mode</b> the system set the choice with higher priority. " \
+          "The <b>manual mode</b> it is set when the user selects a choice manually. " \
+          "When change the current choice also his <b>slaves</b> are modified in the system.</p>" \
+          "<p>To change the current choice of an alternative, select the alternative, click the " \
+          "edit button then select the desired choice in the table and click on the set choice " \
```

Comment:
it is good practice to emphasize name of buttons. I think in YaST we use `<b>` for it.